src/init/application/templates/polymer-2.x/test/_element/_element_test.html

```diff
@@ -22,7 +22,7 @@
       suite('<%= elementName %>', function() {
         test('instantiating the element works', function() {
           var element = fixture('basic');
-          assert.equal(element.is, '<%= elementName %>');
+          assert.equal(element.constructor.is, '<%= elementName %>');
```

Comment:
Thinking about it, a better example might be something like this

```
assert.equal(element.$.heading.textContent, 'Hello World!');
```

and adding the `prop1` to the fixture, and giving the `<h2>` in the element an id of `heading`